src/main/tools/BUILD

```diff
@@ -60,12 +60,6 @@ cc_binary(
         "//src/conditions:windows": ["build-runfiles-windows.cc"],
         "//conditions:default": ["build-runfiles.cc"],
     }),
-    linkopts = select({
```

Comment:
Can you move this `linkopts` to https://github.com/bazelbuild/bazel/blob/0a23bd3d294eebf26ab735a5202504bbba2d641f/src/main/native/windows/BUILD#L18 instead of deleting it? Then you can remove `#pragma comment(lib, "advapi32.lib") //RegGetValueW` in file.h.
This will make it a bit easier to figure out what system libraries are required for the cc_library.

Reply:
Sure, I can do it like that instead. For context, I changed it to this style because the dependency would need to be listed twice (in Bazel and in compile.sh).

Reply:
I see, thanks!